scripts/common/kubernetes.sh

```diff
@@ -733,12 +733,13 @@ metadata:
 spec:
   restartPolicy: OnFailure
   containers:
-  containers:
   - name: pod
     image: $KURL_UTIL_IMAGE
     command: [/usr/local/bin/network, --client, --address, http://kurlnet.default.svc.cluster.local:8080]
 EOF
 
+    spinner_until 120 kubernetes_pod_started kurlnet-client default
```

Comment:
I am personally of the opinion that every spinner needs a log line before it saying what we're waiting for